src/test/java/com/google/sps/IsValidGameNameTest.java

```diff
@@ -67,7 +67,7 @@ public void noGames() {
 
     userList.add(userEntity2);
```

Comment:
is userEntity2 the only thing being added to the userList? If so maybe the arrayList is not needed?

Reply:
yeah, I don't need that I got rid of it, thanks!